### ISSUE TITLE
glibc 2.34 updates break Container Images

### ISSUE THREAD

**Issue by SchoolGuy** [actions/virtual-environments#4297]
### Description

The Ubuntu Hosts have a broken implementation of Docker which is in some operating systems already fixed through patches. The problem is glibc 2.34 which introduced a new syscall.

- openSUSE Bugzilla: https://bugzilla.suse.com/show_bug.cgi?id=1190670
- Ubuntu Launchpad "Fix": https://launchpad.net/ubuntu/+source/glibc/2.34-0ubuntu3
- Workaround found in the community: https://github.com/stanislavlevin/tox-console-scripts/commit/9b2d6e7f1fc9414fc965a2679afddb2fccf1d698

### Virtual environments affected

- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [X] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11
- [ ] Windows Server 2016
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

Virtual Environment
  Environment: ubuntu-20.04
  Version: 20211004.1
  Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20211004.1/images/linux/Ubuntu2004-README.md
  Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20211004.1

- https://github.com/cobbler/cobbler/pull/2812/checks?check_run_id=3871779364
- https://github.com/cobbler/cobbler/pull/2416/checks?check_run_id=3905996345

Examples:

```
make: /bin/sh: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
```

or 

```
---> Running in 8a2750d6d692
terminate called after throwing an instance of 'std::system_error'
  what():  Operation not permitted
```

### Is it regression?

20210919.1

### Expected behavior

Docker Containers can be used normally

### Actual behavior

Syscall is blocked by the Host.

### Repro steps

1. Add a GitHub Actions Workflow which uses Docker
2. Run an openSUSE Tumbleweed Image and try to use `zypper` for example.
3. See error

**Comment by catthehacker** [comment]
Duplicate of https://github.com/actions/virtual-environments/issues/4193
